### DISCUSSION TITLE
AI聊天機器人究竟有多厲害？BBC記者對話ChatGPT

### DISCUSSION THREAD

**Discussion by Xuewen628** [1101-GS3073-C/Python-Community-GS3073-C#635] [General]
對機器人能進行對話的想像在幾十年前就萌生在人類的心中了
看到這真的實現了對我們來說也可說是一種浪漫
只是讓我們開始擔憂未來若機器人真的有自己的思想
那人類該給他們一個什麼樣的社會地位？
https://www.bbc.com/zhongwen/trad/science-63903800

**Comment by awehao** [comment]
我覺得跟機器人聊天，可以得到想要的回覆或是有用的建議，在現在看來稀鬆平常，但是我覺得要更深度更有溫度的談話，還是只能跟人吧，若是機器人有情感了，那應該很可怕吧

**Comment by manipulateed** [comment]
以現在的技術AI還沒有擺脫，智慧侷限在人類給予的資料訓練中，他們還無法創造屬於自己的智慧，那個時代真的會來臨嗎，我們都可能被AI
超越，他們的速度和資料量，以及無所顧忌，都還是人會害怕的東西呢。

**Comment by boson13579** [comment]
我認為ChatGpt不會有自己的思想，因為他們是按照我們餵給他的資料，分類並歸納再回饋資訊給我們這些提問者，要想培養出獨立思想個體還是有一定的難度，至少以目前技術不可行。
